modules/web/src/assets/css/global/_theme.scss

```
@@ -298,6 +298,10 @@
     background-color: transparent;
   }
 
+  .mat-dialog-content {
+    border-bottom: 1px solid map.get($colors, dialog-action-separator);
```

IMO we should use the `divider` color which we use in other places for borders. Wdyt?
```suggestion
    border-bottom: 1px solid map.get($colors, divider);
```